quantities/tests/test_umath.py

```diff
@@ -293,3 +293,13 @@ def test_greater_equal(self):
         arr2 = (1.0, 2.0) * pq.m
         self.assertTrue(np.all(np.greater_equal(arr2, arr1)))
         self.assertFalse(np.all(np.greater_equal(arr2*0.99, arr1)))
+
+    def test_maximum(self):
+        arr1 = (998, 999) * pq.m
+        arr2 = (1e3, 5e2) * pq.m
+        self.assertQuantityEqual(np.maximum(arr1, arr2) - [1000, 999]*pq.m, [0, 0]*pq.m)
```

Comment:
Out of curiosity why do the subtraction and then compare to 0 rather than compare directly to the values? Is there a benefit to doing this is readability or reliability? :)

Reply:
Just a habit of mine, no particular reason.